README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
   <a href="https://vuestic.dev" target="_blank">
-    <img alt="Vuestic UI Logo" width="500" src="./public/img/vuestic-admin-logo.png">
```

Comment:
A added the vuestic logo in assets in purpose. So that it's a part of the bundle and not served separately. If you think that it's better to serve it from the public folder, make sure to remove the duplicate from assets.